tests/hvac_test_utils.py

```diff
@@ -41,6 +41,8 @@ def get_random_config_data(topic):
     """
     data = {}
     # Retrieve the config items of the topic.
+    xml = MqttInfoReader()
+    xml.collect_hvac_topics_and_items_from_json()
```

Comment:
You use this a lot. Is Please consider making the class smart enough to call collect_hvac_topics_and_items_from_json itself, if needed. Clearly the user only cares about the `get_...` method.

That makes the class easier to use correctly, as there is no worry of forgetting to call this "method that must be called before get_...".

Reply:
I was also considering this but probably forgot to do it because of all the other changes that I made. Done.